sql/src/main/java/org/apache/druid/sql/calcite/rule/DruidRules.java

```diff
@@ -118,8 +118,7 @@ public static List<RelOptRule> rules(PlannerContext plannerContext)
     if (plannerContext.featureAvailable(EngineFeature.UNNEST)) {
       retVal.add(new DruidUnnestRule(plannerContext));
       retVal.add(new DruidCorrelateUnnestRule(plannerContext));
-      retVal.add(ProjectCorrelateTransposeRule.INSTANCE);
-      retVal.add(CorrelateFilterLTransposeRule.instance());
```

Comment:
is CorrelateFilterLTransposeRule not needed?

Reply:
No it was a custom rule we were creating, it is not needed anymore